### PR TITLE
updated arduino c example for v 4.3.1

### PR DESCRIPTION
**Pull request by rherrmannr**

### CODE REVIEW

File: org.yakindu.sct.examples.trafficlight.c.arduino/.cproject [hunk 1]
```diff
@@ -12,52 +12,59 @@
...
+							</tool>
+							<tool id="io.sloeber.tool.ar.1942578340" name="Arduino archiver" superClass="io.sloeber.tool.ar"/>
+							<tool id="io.sloeber.tool.combine.1791009770" name="Arduino combiner" superClass="io.sloeber.tool.combine"/>
+							<tool id="io.sloeber.tool.objcopy.eep.2005776796" name="Arduino tool objcopy command" superClass="io.sloeber.tool.objcopy.eep"/>
+							<tool id="io.sloeber.tool.size.1574078578" name="Arduino tool Print Size" superClass="io.sloeber.tool.size"/>
 						</toolChain>
 					</folderInfo>
 					<sourceEntries>
-						<entry excluding="libraries/**/?xamples/**|libraries/**/?xtras/**|libraries/**/test/**|libraries/**/third-party/**|libraries**/._*" flags="VALUE_WORKSPACE_PATH" kind="sourcePath" name=""/>
+						<entry excluding="libraries/?*/**/?xamples/**|libraries/?*/**/?xtras/**|libraries/?*/**/test*/**|libraries/?*/**/third-party/**|libraries/**/._*|libraries/?*/utility/*/*" flags="VALUE_WORKSPACE_PATH" kind="sourcePath" name=""/>
```
Comment (tangele) [inline org.yakindu.sct.examples.trafficlight.c.arduino/.cproject]: `?*` looks wrong. Not sure if i missunderstand something, but for me that excludes things like `abcd` and only allows `aa`, `bbb`, etc.

File: org.yakindu.sct.examples.trafficlight.c.arduino/.cproject [hunk 2]
```diff
@@ -12,52 +12,59 @@
...
 					</sourceEntries>
 				</configuration>
 			</storageModule>
 			<storageModule moduleId="org.eclipse.cdt.core.externalSettings"/>
 		</cconfiguration>
 	</storageModule>
 	<storageModule moduleId="cdtBuildSystem" version="4.0.0">
-		<project id="traffic_light_arduino.null.1229072795" name="traffic_light_arduino"/>
+		<project id="prjkt.null.802440666" name="prjkt"/>
```
Comment (tangele) [inline org.yakindu.sct.examples.trafficlight.c.arduino/.cproject]: `prjkt` looks accidential. Is that desired?

File: org.yakindu.sct.examples.trafficlight.c.arduino/.cproject (hunk 1 above)
Comment (tangele) [inline org.yakindu.sct.examples.trafficlight.c.arduino/.cproject]: `?*` looks wrong. Not sure if i missunderstand something, but for me that excludes things like `abcd` and only allows `aa`, `bbb`, etc.

File: org.yakindu.sct.examples.trafficlight.c.arduino/.cproject (hunk 2 above)
Comment (tangele) [inline org.yakindu.sct.examples.trafficlight.c.arduino/.cproject]: `prjkt` looks accidential. Is that desired?

File: org.yakindu.sct.examples.trafficlight.c.arduino/index.html [hunk 3]
```diff
@@ -39,18 +39,25 @@ <h2>Installing the Arduino Plugin</h2>
...
-        One plugin that works pretty well can be found at <a
-          href="http://eclipse.baeyens.it/">http://eclipse.baeyens.it/</a>. In
-        YAKINDU SCT, simply install it by selecting <em>Help → Install New
-          Software</em> in the main menu, typing or pasting
-        <code>http://www.baeyens.it/eclipse/V3</code>
-        into the proper field, and follow the regular procedure. SCT will
-        restart and download the necessary Arduino libraries for you.
-      </p>
+        <p> 
+        	The easiest way of how to install the plugin is using the example wizard.
```
Comment (tangele) [inline org.yakindu.sct.examples.trafficlight.c.arduino/index.html]: `The easiest way of how to install` --> `The easiest way to install`

File: org.yakindu.sct.examples.trafficlight.c.arduino/index.html [hunk 4]
```diff
@@ -39,18 +39,25 @@ <h2>Installing the Arduino Plugin</h2>
...
+        	Select the 'Traffic Light (C) for Arduino' example, click on 
+        	'Install Dependencies...' and follow the instructions.
+        	Your IDE may restart. SCT will
+        	restart and download the necessary Arduino libraries for you.
+        </p>
+        	Another way is using the update side, which can be found at <a 
+        	href="http://eclipse.baeyens.it/">http://eclipse.baeyens.it/</a>. In
+        	YAKINDU SCT, simply install it by selecting <em>Help → Install New
+        	Software</em> in the main menu, typing or pasting
+        	<code>http://www.baeyens.it/eclipse/V4</code>
```
Comment (tangele) [inline org.yakindu.sct.examples.trafficlight.c.arduino/index.html]: update link, /stable/ is missing.

File: org.yakindu.sct.examples.trafficlight.c.arduino/.cproject (hunk 1 above)
Comment (rherrmannr) [reply org.yakindu.sct.examples.trafficlight.c.arduino/.cproject]: this is auto generated from a third party plug-in

**Pull request merged**